README.md

```diff
@@ -86,8 +86,8 @@ process.mainArgs = process.argv.slice(process._exec ? 1 : 2)
     * `short` {string} (Optional) A single character alias for an option; When appearing one or more times in `args`; Respects the `multiple` configuration
   * `strict` {Boolean} (Optional) A `Boolean` on wheather or not to throw an error when unknown args are encountered
 * Returns: {Object} An object having properties:
-  * `flags` {Object}, having properties and `Boolean` values corresponding to parsed options passed
-  * `values` {Object}, have properties and `String` values corresponding to parsed options passed
+  * `foundOptions` {Object}, key:true for each option found in the input `args`
+  * `values` {Object}, key:value for each option found. Value is a string for string options, or `true` for boolean options, or an array for  options configured as `multiple:true`.
```

Comment:
```suggestion
  * `values` {Object}, key:value for each option found. Value is a string for string options, or `true` for boolean options, or an array of strings for options configured as `multiple:true`.
```

Reply:
It is an array of `true` for a boolean option marked as `multiple:true`. 

Maybe:
- or an array of these for options configured as `multiple:true`.
- or an array of string/`true` for options configured as `multiple:true`.

Reply:
I'm confused; how can you have multiple boolean values on the same flag? `--foo --foo` would be `[true, true]`? What's the use case for that?

To me, "multiple" has no meaning on a boolean-valued option, and configuring that should be an error.

Reply:
The boolean use case is for counting:

```js
const { parseArgs } = require('@pkgjs/parseargs');

const { foundOptions, values } = parseArgs({ options: { 
    verbose: { short: 'v', multiple: true }
}});
const verbosity = foundOptions.verbose ? values.verbose.length : 0;
console.log(`Verbosity is ${verbosity}`);
```

```
% node verbosity.js     
Verbosity is 0
% node verbosity.js -vvv
Verbosity is 3
```

Reply:
Great example 👍 I've definitely used this from time to time while debugging remote git branchs: e.g.
`git branch -v` and `git branch -vv`

Reply:
Oof, ok, this seems super gross and obscure, but fair enough